### DISCUSSION TITLE
Moniter gets freezed when i start up isaaclab scripts.

### DISCUSSION THREAD

**Discussion by Omikaa** [isaac-sim/IsaacLab#555] [Q&A]
I run the script using 'python source/standalone/workflows/sb3/play.py --task Isaac-Quadcopter-Direct-v0 --num_envs 1 --use_last_checkpoint'. 
While the isaacsim's window is opened, my external moniter is freezed.
However, I try to change the command with 'python source/standalone/workflows/sb3/play.py --task Isaac-Quadcopter-Direct-v0 --num_envs 1 --use_last_checkpoint **--headless** '. The moniter works fine. 
Can someone help me solve the problem? Thanks very much!

**Comment by Mayankm96** [comment]
Does this happen with other standalone demo scripts?

**Reply by Omikaa** [reply]
The problem is solved out. I found that it is the conflict between integrated vedio card and external vedio card. Because I am working with laptop.
So I change the mode of display to connect directly external graphics card. No more problems come out.
